docs/configuration.md

```diff
@@ -585,13 +586,18 @@ It uses configuration from `exchange.pair_whitelist` and `exchange.pair_blacklis
 
 #### Volume Pair List
 
-`VolumePairList` selects `number_assets` top pairs based on `sort_key`, which can only be `quoteVolume`.
+`VolumePairList` employs sorting/filtering of pairs by their trading volume. I selects `number_assets` top pairs with sorting based on the `sort_key` (which can only be `quoteVolume`).
+
+!!! Deprecation
```

Comment:
I'm not sure I properly declared the deprecation warning block here, need to search if we have other similar notes in the docs...

Reply:
This will not work.

we got the link to the documentation in the developer settings (or you can use [this one](https://squidfunk.github.io/mkdocs-material/extensions/admonition/#types) )- which lists available types (deprecation is not one of these).

honestly, i'd just keep this section out - i've intentionally not documented this here as i find "deprecated" documentation confusing for new users. 

what's that mean? can i use it - or better not? ... see what i mean?